### PR TITLE
ADD c and m as accepted payload attributes

### PR DESCRIPTION
**Pull request by dmoranj**
Accept c and m along with cadena as payload attributes.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,4 +1,5 @@
 - ADD Logops instead of Fiware-node-logger (#44).
 - ADD Decode result of the Black Button in the IOTA (#64).
 - FIX RPM upgrade overrides config file (#69).
-- ADD Computation of the Humidity percentage (#72).
\ No newline at end of file
+- ADD Computation of the Humidity percentage (#72).
+- ADD C and M as accepted payload attributes in request.
```
Comment (gtorodelvalle) [inline CHANGES_NEXT_RELEASE]: Final blank line appreciated... ;)

**Comment by gtorodelvalle** [comment]
LGTM on my side apart from a minor comment... :)

**Comment by dmoranj** [comment]
Changed also the success return code to 1 instead of the received action.

**Comment by gtorodelvalle** [comment]
LGTM and merging! ;) Thanks @dmoranj :)

**Pull request merged**